### ISSUE TITLE
Campaign sticker

### ISSUE THREAD

**Issue by TrollerCoasterGR** [Lurkars/gloomhavensecretariat#432]
### Describe the bug

We recently unlocked the "Statue" campaign sticker by reaching morale 20 through an outpost event but we can't add it to our campaign stickers.

### Game Enviroment

Frosthaven

### Expected behavior

There should be the coresponding campaign sticker to be added manually.

### Gloomhaven Secretariat Version

v0.80.4

### Used Browser

Chrome 119.0.6045.106 (64-bit)

### OS

Windows 11

**Comment by kimmathiassen** [comment]
As I remember the Statue go on the town map, not the campaign sheet. 
![image](https://github.com/Lurkars/gloomhavensecretariat/assets/2360557/fd920ed9-c44c-4861-8302-304cab52256c)

**Comment by Lurkars** [comment]
I have no access to a physical FH copy yet. So all my data comes from public resources like Books as PDFs and the Worldhaven Repository. Therefore I got all available **Campaign Stickers** just from that source: https://github.com/any2cards/worldhaven/tree/master/images/art/frosthaven/stickers/individual/campaign-stickers 

Looking at the available stickers, the **Statue** is listed there under **Outpost Stickers**. So we now got in a strange situation, because you're right that **161.3** is talking about **Campaign Sticker**, but also to put it on the Map (as @kimmathiassen already mentioned). From just simple POV, it's not placed on the Campaign Sheet and therefor current behavior is correct. On the other hand I can understand, that you want to view that rewards tracked in GHS somehow too. So I would just asked here, what's you're preferred way. The real correct solution would be a complete and correct implementation of #421, that would just place that sticker on the map and you're done. So if there is no hurry, I would close this issue with the reference of #421, but must admin that I have no idea, when this really will find it's way into GHS, because it's a ton of manual work to create the map...

**Comment by TrollerCoasterGR** [comment]
You are correct, the corresponding section does say Campaign Sticker so tried to add it to the app's Campaign Sticker section but at the end we just put it on the map. We are really looking forward for the event management feature to become functional, it would be a huge quality of life improvement. Also, there should be a Notes section on the Campaign Sheet just to be able to note general things regarding the Campaign and/or the party in general. Thank you for the great job you've done so far!

**Comment by TrollerCoasterGR** [comment]
It is visible on the map. But when I did the update the app tracked our morale to 20 automatically but it was currently at 16.

**Comment by Lurkars** [comment]
I am pretty sure you miss-clicked somehow or so. Anyway, as said I will update to automatically apply section and then we can take a deeper look and test again.

**Comment by Lurkars** [comment]
Currently no support for events, I will take a look, how I can add support for this event anyhow.

**Comment by jjanvier** [comment]
@Lurkars by any chance, did you solve the problem mentioned by @ronderksen ?
